docs/administration/security/outbound-requests.md

```diff
@@ -50,15 +50,41 @@ In addition, if you also check the box "Help improve Octopus by sending usage st
 4. The number of each of the different types of deployment steps that you have
 5. The number of releases and deployments that you have done in the last 90 days
 6. The number of users that you have
-7. (As of 2.6) A SHA1 hash of your license key serial number
-8. (As of 2.6) The amount of memory Octopus is using
-9. (As of 2.6) The % of CPU Octopus is using on average
-10. (As of 3.12) The time to your first deployment
+
```

Comment:
Maybe make this a table, rather than headings?

Or should this be partials, based on the version?

Reply:
Once docs are snapshotted we can just edit this rolling forward and trust the archives to be accurate for old versions. For now, I might follow your other suggestion of making it a table and then we can snapshot it off this month, tidy it up and power to the future. I'll amend this shortly